bigquery/docs/snippets.py

```diff
@@ -1327,35 +1255,6 @@ def test_extract_table_compressed(client, to_delete):
     to_delete.insert(0, blob)
 
 
-def test_client_query_legacy_sql(client):
```

Comment:
OK to remove. I see we have this sample at https://github.com/googleapis/google-cloud-python/blob/master/bigquery/samples/client_query_legacy_sql.py